src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -265,6 +265,10 @@ ov::SupportedOpsMap Plugin::query_model(const std::shared_ptr<const ov::Model>&
             transform_model(model, config);
         },
         [&prog](std::shared_ptr<ov::Node> node) {
+            for (size_t i = 0lu; i < node->get_output_size(); i++) {
+                if (node->get_output_element_type(i) == ov::element::string)
```

Comment:
I added a bit more accurate precision check here https://github.com/openvinotoolkit/openvino/pull/23860
So I think this change is not need anymore

Reply:
Removed